spec/core/flow_spec.rb

```diff
@@ -5,35 +5,38 @@
 
 describe Flow do
 
+    let(:ui) { UI.new }
     let(:board) { Board.new(3) }
-    let(:output) { StringIO.new }
-
-  it 'can not place a symbol if the cell is not empty' do
-    user_input = StringIO.new("1\n1\n2\n3\n4\n5\n6\n7\n8\n9")
-    ui = UI.new(output, user_input)
-    flow = Flow.new(ui, board)
-    flow.start
-    expect(output.string).to include("Please choose a different position where the cell is empty\n")
-  end
+    let(:flow) { Flow.new(ui, board) }
+
+    def mark_board(marks)
+      for position in marks
+        board.mark(position, "X")
+      end
+    end
 
   it 'current player is the first player in the array' do
-    ui = UI.new
-    flow = Flow.new(ui, board)
     expect(flow.current_player.symbol).to eq("X")
   end
 
   it 'opponent is the last player in the array' do
-    ui = UI.new
-    flow = Flow.new(ui, board)
     expect(flow.opponent.symbol).to eq("O")
   end
 
   it 'swap players in the array' do
-    ui = UI.new
-    flow = Flow.new(ui, board)
     flow.swap_players
     expect(flow.current_player.symbol).to eq("O")
     expect(flow.opponent.symbol).to eq("X")
   end
 
+  it 'game is over' do
+    mark_board([0, 1, 2])
+    expect(flow.game_over).to be true
```

Comment:
Be consistent about the parenthesis usage for the be/eq (also with be vs. eq)

Reply:
is it the same? when do you use be vs eq? 
I will put the parenthesis now, thanks!!

Reply:
Personally, never used `be`, always eq